backend-external/src/utils.ts

```diff
@@ -1,10 +1,31 @@
 import { NextFunction, Request, Response } from 'express';
 import axios, { AxiosInstance } from 'axios';
 import { config } from './config';
+import { Options } from 'swagger-jsdoc';
+
 const backendAxios: AxiosInstance = axios.create({
   baseURL: config.get('backend:url'),
-  headers: { 'x-api-key': config.get('backend:apiKey') }
+  headers: { 'x-api-key': config.get('backend:apiKey') },
 });
+
+const swaggerOpenAPIOptions: Options = {
+  definition: {
+    openapi: '3.1.0',
+    info: {
+      title: 'Pay Transparency External API',
+      version: '1.0.0',
+      description:
+        'These api exposes endpoint to query pay transparency reports',
+    },
+    servers: [
+      {
+        url: 'http://localhost:3002/api/v1/pay-transparency',
```

Comment:
can we pass env var here, and replace it with actual route url from helm charts something like `URL`
refer here https://github.com/bcgov/fin-pay-transparency/blob/main/charts/fin-pay-transparency/charts/backend-external/templates/route.yaml#L12

Reply:
I will need your help with this on Monday @sukanya-rath